apps/web/src/styles/printStyles.ts

```diff
@@ -2,4 +2,29 @@
 export default `
 @page {
   size: letter portrait;
-}`;
+}
+
+h1, h2, h3, h4, h5 {
+  page-break-after: avoid;
+}
+
+h1 {
+  font-size: 22px;
+}
+
+h2 {
+  font-size: 18px;
+}
+
+h3 {
+  font-size: 15px;
+}
+
+h4 {
+  font-size: 13px;
+}
+
+p {
+  font-size: 9px;
+}
```

Comment:
The issue mentions points so these will result in the incorrect size. I think it should be:

- H1 - `1.8331rem`
- H2 - `1.5rem`
- H3 - `1.25rem`
- H4 - `1.0831rem`
- H5 - `0.9169rem`
- H6 - `0.8331rem`
- P - `0.75rem`

Reply:
Fixed by [6617bce](https://github.com/GCTC-NTGC/gc-digital-talent/pull/7747/commits/6617bce2da34effbf1e1babd5c0e982fea963377)

Reply:
Oh my, I just noticed that setting `p` makes it look weird next to items that are not in any of these tags. I wonder if we just use this instead and just use `pt` directly instead of messing around with `rem` 😢 I'm sorry, this is 💯 my fault!

```css
html, body {
  font-size: 9pt !important;
}

h1 {
  font-size: 22pt !important;
}

h2 {
  font-size: 18pt !important;
}

h3 {
  font-size: 15pt !important;
}

h4 {
  font-size: 13pt !important;
}

h5 {
  font-size: 12pt !important;
}

h6 {
  font-size: 10.5pt !important;
}
```